packages/cli-repl/src/cli-repl.ts

```diff
@@ -51,7 +53,28 @@ class CliRepl {
 
     this.serviceProvider = await CliServiceProvider.connect(driverUri, driverOptions);
     this.ShellEvaluator = new ShellEvaluator(this.serviceProvider, this.bus, this);
-    this.mdbVersion = await this.serviceProvider.getServerVersion();
+    this.buildInfo = await this.serviceProvider.buildInfo();
+
+    let cmdLineOpts = null;
```

Comment:
I'd really extract this whole new block in a method.

Reply:
Wouldn't I have to wrap the new block in another try/catch? Or do you mean an async block that returns object or null?

Reply:
ah nvm!